src/main/java/core/basesyntax/service/RegistrationService.java

```diff
@@ -3,5 +3,5 @@
 import core.basesyntax.model.User;
 
 public interface RegistrationService {
-    User register(User user);
+    User register(User user) throws UserRegistrationException;
```

Comment:
Если твоё исключение наследуется от RuntimeException - значит оно непроверяемое (Unchecked). 
Такое исключение не нужно указывать в сигнатуре метода (throws)
```suggestion
    User register(User user);
```